src/main/kotlin/me/partlysanestudios/partlysaneskies/PartlySaneSkies.kt

```diff
@@ -221,13 +222,28 @@ class PartlySaneSkies {
         registerEvent(EventManager)
         val playerRating = PlayerRating() // Kotlin object supremacy
         registerEvent(playerRating)
+        val skymartValue = SkymartValue()
+        registerEvent(skymartValue)
+        val compostValue = CompostValue()
+        registerEvent(compostValue)
+        val mathematicalHoeRightClicks = MathematicalHoeRightClicks()
+        registerEvent(mathematicalHoeRightClicks)
+
 
         // Registers all Partly Sane Skies Events
+        EventManager.register(RequiredSecretsFound())
+        EventManager.register(MinionData())
+        EventManager.register(SkyblockDataManager())
         EventManager.register(DebugKey)
         EventManager.register(TerminalWaypoints)
-        EventManager.register(playerRating)
         EventManager.register(PearlRefill)
-        EventManager.register(RequiredSecretsFound())
+        EventManager.register(TreecapitatorCooldown)
+        EventManager.register(WrongToolCropWarning)
+        EventManager.register(WrongToolCropWarning.CropToolData)
+        EventManager.register(playerRating)
```

Comment:
Okay so how about extending on the loader and potentially loading the eventmanager in there aswell?

Reply:
> about extending on the loader

?

Reply:
The registerEvent() thhing

Reply:
Done